jwst/combine_1d/combine1d.py

```diff
@@ -619,64 +618,53 @@ def check_exptime(exptime_key):
     return exptime_key
 
 
-def do_correction(asn_file, exptime_key, interpolation, background=False):
+def do_combine(input_model, exptime_key, background=False):
```

Comment:
Sorry to be picky but this function is general enough that it can probably be used on its own by users. In which case the name `do_combine` is too general. Can we make it more specific, e.g. `combine_spectra_1d`. I'm open to other descriptive names.

Reply:
I changed the function name to `combine_1d_spectra`.